### PR TITLE
Conscious language improvements

### PR DESCRIPTION
**Pull request by droideck**

### CODE REVIEW

File: docs/389ds/FAQ/esc-guide.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The Dogtag Certificate System creates, manages, renews, and deletes certificates
 -   The Data Recovery Manager (DRM), which archives and recovers keys.
 -   The Online Certificate Status Manager, which stores lists of revoked certificates for client applications to use to check if a certificate is valid.
 -   The Token Processing System (TPS), which interacts with smart cards to generate and store keys and certificates for a specific user.
--   The Token Key Service (TKS), which generates and stores master keys used by the TPS.
```
Comment (tbordaz) [inline docs/389ds/FAQ/esc-guide.md:30]: Are you sure we can change this word. Someone from PKI should confirm

File: docs/389ds/FAQ/faq.md [hunk 2]
```diff
@@ -117,25 +117,25 @@ See [History](../FAQ/history.html) for a history of Netscape, iPlanet, and Sun D
...
 
-### How does the 389 multi-master replication work?
+### How does the 389 multi-supplier replication work?
 
-389 replication uses a push model. A replication supplier is a server that pushes updates to other servers. A master is a supplier that can receive update requests from clients. A hub is a supplier that receives update requests from other replication suppliers and pushes updates to other replicas.
+389 replication uses a push model. A replication supplier is a server that pushes updates to other servers. A supplier is a supplier that can receive update requests from clients. A hub is a supplier that receives update requests from other replication suppliers and pushes updates to other replicas.
```
Comment (tbordaz) [inline docs/389ds/FAQ/faq.md:122]: Would change it into: A supplier is a LDAP server that can receive and process update requests from clients.

File: docs/389ds/FAQ/selinux-policy.md [hunk 3]
```diff
@@ -75,7 +75,7 @@ When the service command is used, **ldap-agent** will use **/etc/dirsrv/config/l
 
 #### Port Access
 
-The subagent does not listen on any ports itself. The sub-agent simply communicates with the snmpd master agent, which accesses the ports itself.
+The subagent does not listen on any ports itself. The sub-agent simply communicates with the snmpd supplier agent, which accesses the ports itself.
```
Comment (tbordaz) [inline docs/389ds/FAQ/selinux-policy.md:78]: I would say 'main' agent as it is not related to replication

File: docs/389ds/FAQ/update-resolution-for-single-valued-attributes.md [hunk 4]
```diff
@@ -231,20 +231,20 @@ DISCUSSION TOPIC: Do we need to handle pending values at all ? Assume a mod of a
 Concurrent state changes
 ========================
 
-There are many operations which lead to state changes in a SVA, they can be applied in any combination and order and on a wide range of masters.
+There are many operations which lead to state changes in a SVA, they can be applied in any combination and order and on a wide range of suppliers.
 
-To determine the correct behaviour all combinations in a two master topology are considered. If more masters are involved the variety increases, but in URP there will always be only the resolution of a current state with a state change received via replication.
+To determine the correct behaviour all combinations in a two supplier topology are considered. If more suppliers are involved the variety increases, but in URP there will always be only the resolution of a current state with a state change received via replication.
```
Comment (tbordaz) [inline docs/389ds/FAQ/update-resolution-for-single-valued-attributes.md:236]: two suppliers

File: docs/389ds/FAQ/update-resolution-for-single-valued-attributes.md [hunk 5]
```diff
@@ -231,20 +231,20 @@ DISCUSSION TOPIC: Do we need to handle pending values at all ? Assume a mod of a
...
+So systematically all combinations involving two suppliers are investigated, for the test setup an extended topology with three suppliers and one consumer is used to verify consistency and expected states and values
 
-Two master topology
+Two supplier topology
 -------------------
 
-The following table lists all the combinations of state changes in two master topology, each state change has to be executed with the potential operations to achieve this state change. It is assumed that the CSN of the state change on master 1 is always lower than that on master 2. This is also the reason that the table contains the full combinations since (--\>S0,--\>S1) and (--\>S1,--\>S0) are not symmetric.
+The following table lists all the combinations of state changes in two supplier topology, each state change has to be executed with the potential operations to achieve this state change. It is assumed that the CSN of the state change on supplier 1 is always lower than that on supplier 2. This is also the reason that the table contains the full combinations since (--\>S0,--\>S1) and (--\>S1,--\>S0) are not symmetric.
```
Comment (tbordaz) [inline docs/389ds/FAQ/update-resolution-for-single-valued-attributes.md:243]: two suppliers

File: docs/389ds/FAQ/update-resolution-for-single-valued-attributes.md [hunk 6]
```diff
@@ -264,12 +264,12 @@ The operations to create the state changes are different, but results should be
...
 
-Three master topology
+Three supplier topology
 ---------------------
 
-Although with more than two masters the update resolution procedure should sequentially applied and result in the same state resolution existing bugs show that this is not the case with the current implementation. This chapter will handle all scenarios with 3 masters, this also allows all three different state changes applied in parallel. An extension to more than three master should not be necessary. In the tests a consumer should be added to the topology, it serves as a reference where the modifications are applied in the csn order.
+Although with more than two suppliers the update resolution procedure should sequentially applied and result in the same state resolution existing bugs show that this is not the case with the current implementation. This chapter will handle all scenarios with 3 suppliers, this also allows all three different state changes applied in parallel. An extension to more than three supplier should not be necessary. In the tests a consumer should be added to the topology, it serves as a reference where the modifications are applied in the csn order.
```
Comment (tbordaz) [inline docs/389ds/FAQ/update-resolution-for-single-valued-attributes.md:270]: three suppliers

File: docs/389ds/design/dna-plugin-proposal.md [hunk 7]
```diff
@@ -33,37 +33,37 @@ Mighty ASCII art with 4 servers and only SRV 1 creating users, each server has o
...
+Manually splitting the ID space into ranges would be easy. And assigning a range to each server too. The advantage of using a range is that if one server is running low on IDs all is needed is only one other supplier available that has a big enough range assigned. The 2 servers can in fact be reconfigured so that half of the range previously assigned to one server can be reassigned to the other without the need for other suppliers to be online. The change of ownership information will be replicated to them as soon as they come up. There is no possibility that one supplier will steal ranges from another one as communication between the new and the old owner is required to allow the transfer of ownership.
```
Comment (tbordaz) [inline docs/389ds/design/dna-plugin-proposal.md:48]: I have a doubt here with the terminology of the DNA plugin. With DNA a server can request range to another server. I do not know how it was express in DNA plugin description. We should stick to what is in the doc

File: docs/389ds/design/dna-plugin-proposal.md [hunk 8]
```diff
@@ -106,5 +106,5 @@ The first operation will fail if the range has already been manipulated by someo
 
 ### Considerations
 
-What happen if the server is directly reachable but the replication topology is broken and the changes do not reach back the original server ? After the first change the plugin will have outdated information, when the grace period expires the plugin will, probably, try again with the same server and fail to the operation to grab a new range, at this point it will switch to another server. The Master requesting the new ranges cannot use them until the replication is over and its db has been changed accordingly to inform it of the new ownership.
+What happen if the server is directly reachable but the replication topology is broken and the changes do not reach back the original server ? After the first change the plugin will have outdated information, when the grace period expires the plugin will, probably, try again with the same server and fail to the operation to grab a new range, at this point it will switch to another server. The Supplier requesting the new ranges cannot use them until the replication is over and its db has been changed accordingly to inform it of the new ownership.
```
Comment (tbordaz) [inline docs/389ds/design/dna-plugin-proposal.md:109]: The same, we should use the terminology that is in the DNA plugin documentation

File: docs/389ds/design/managing-repl-conflict-entries.md [hunk 9]
```diff
@@ -10,15 +10,15 @@ title: "Managing Replication Conflict Entries"
 Overview
 ========
 
-In multimaster replication the same entry (entries with the same DN), can be added concurrently and will be accepted by the masters whhic handle the ADD. After replication there would be two entries with the same DN, which violates the LDAP data model. So replication needs to “resolve this conflict.
+In multisupplier replication the same entry (entries with the same DN), can be added concurrently and will be accepted by the suppliers whhic handle the ADD. After replication there would be two entries with the same DN, which violates the LDAP data model. So replication needs to “resolve this conflict.
```
Comment (tbordaz) [inline docs/389ds/design/managing-repl-conflict-entries.md:13]: Here a typo. 'whhic' -> 'which'

File: docs/389ds/howto/howto-bind.md [hunk 10]
```diff
@@ -57,7 +57,7 @@ To implement this solution, the following things need to happen:
         dnszonename: foobar.com
         dnsclass: IN
         dnstype: SOA
-        dnszonemaster: tim.foobar.com
+        dnszonesupplier: tim.foobar.com
```
Comment (tbordaz) [inline docs/389ds/howto/howto-bind.md:60]: I have a doubt. Is dnszonemaster a valid term ?

**Review by tbordaz — COMMENTED** [review]
Few comments

File: docs/389ds/howto/howto-bind.md (hunk 10 above)
Comment (progier389) [reply docs/389ds/howto/howto-bind.md:60]: IMHO the whole howto-bind document is outdated, the links to ldap2dns now leads to sourceforge main page but  they do not have any project about ldap2dns ... 
IMHO we should rather left it as it and move it in an "Outdated" section ...

File: docs/389ds/FAQ/esc-guide.md (hunk 1 above)
Comment (droideck) [reply docs/389ds/FAQ/esc-guide.md:30]: I've changed it back as I confirmed with the recent docs that they still use 'master' terminology.

File: docs/389ds/FAQ/update-resolution-for-single-valued-attributes.md (hunk 4 above)
Comment (droideck) [reply docs/389ds/FAQ/update-resolution-for-single-valued-attributes.md:236]: I think it okay as it means "a topology of two suppliers". So it should be two supplier topology.

But I think even better option will be to say: "two-supplier topology". Hence, I'll fix like that.

File: docs/389ds/FAQ/update-resolution-for-single-valued-attributes.md (hunk 5 above)
Comment (droideck) [reply docs/389ds/FAQ/update-resolution-for-single-valued-attributes.md:243]: Same here.

File: docs/389ds/design/dna-plugin-proposal.md (hunk 7 above)
Comment (droideck) [reply docs/389ds/design/dna-plugin-proposal.md:48]: I checked both Admin and Config guides and we use the supplier terminology there.

File: docs/389ds/design/dna-plugin-proposal.md (hunk 8 above)
Comment (droideck) [reply docs/389ds/design/dna-plugin-proposal.md:109]: Same here.

**Comment by droideck** [comment]
Fixed. Please, review...

**Review by tbordaz — APPROVED** [review]
Thanks for your answers. LGTM

**Comment by droideck** [comment]
> Thanks for your answers. LGTM

Thanks for the review!

@progier389 I haven't found "Outdated" section, so I moved it to "Legacy How To's" one. Is it okay with you?

**Review by progier389 — APPROVED** [review]
LGTM

**Pull request merged**